### PR TITLE
Feat/improve datasets

### PR DESCRIPTION
**Pull request by lvwerra**

### CODE REVIEW

File: notebooks/00_core.ipynb [hunk 1]
```diff
@@ -43,20 +43,21 @@
     "    \"\"\"\n",
```
Comment (lewtun) [inline notebooks/00_core.ipynb:43]: <p>Delete this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (lewtun) [inline notebooks/00_core.ipynb:43]: <p>"This dataset from Kaggle ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (lewtun) [inline notebooks/00_core.ipynb:43]: <p>"dataset is a cleaned subset of the <code>housing_gmaps_data_raw.csv</code>  dataset."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (lewtun) [inline notebooks/00_core.ipynb:43]: <p>Maybe add a section for this dataset, similar to what you did with the housing prices?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (lewtun) [inline notebooks/00_core.ipynb:43]: <p>Ditto as IMBD (add section)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by lewtun** [comment]
Overall looks good to me - just make the changes and you can merge.  If you add a keyword like "Closes PR #PR_number" you can automatically close the issue on our project board: https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by lvwerra** [comment]
This PR closes #16 and closes #18 .

**Pull request merged**